server/spec/subscription_resource_spec.rb

```diff
@@ -48,4 +48,42 @@
           e.to_s.eql? "bad URI(is not URI?): pools/{pool_id}/cert"
       end
   end
+
+
+  it 'subscriptions derived/provided products should return empty array when null' do
```

Comment:
Indentation needs to be fixed. It's uneven at multiple places. Same for #3052 PR.

Reply:
Done.